packages/core/src/actions/contracts/multicall.ts

```diff
@@ -70,18 +72,27 @@ export async function multicall<Data extends any[] = Result[]>({
         contractInterface,
       })
       const params = Array.isArray(args) ? args : args ? [args] : []
-      const callData = contract.interface.encodeFunctionData(
-        functionName,
-        params,
-      )
-      if (!contract[functionName])
-        console.warn(
-          `"${functionName}" is not in the interface for contract "${addressOrName}"`,
+      try {
+        const callData = contract.interface.encodeFunctionData(
+          functionName,
+          params,
         )
-      return {
-        target: addressOrName,
-        allowFailure,
-        callData,
+        if (!contract[functionName])
+          console.warn(
+            `"${functionName}" is not in the interface for contract "${addressOrName}"`,
+          )
+        return {
+          target: addressOrName,
+          allowFailure,
+          callData,
+        }
+      } catch (err) {
+        if (!allowFailure) throw err
+        return {
+          target: addressOrName,
+          allowFailure,
+          callData: '0x',
```

Comment:
If we catch an error from encoding function data, we will make the calldata for that call nullish.